crypto/ecdsa.c

```diff
@@ -30,7 +30,6 @@
 #include "address.h"
 #include "base58.h"
 #include "bignum.h"
-#include "ecdsa.h"
```

Comment:
If we remove ecdsa.h, does this still compile successfully without `USE_SECP256K1_ZKP_ECDSA`? In principle code shouldn't rely on another header file including another necessary header file.

Reply:
> If we remove ecdsa.h, does this still compile successfully without USE_SECP256K1_ZKP_ECDSA?

Yes, it does. I think it is is included indirectly by `#include "secp256k1.h"`.

To be honest, I don't remember why I removed the include. I'm reverting it in https://github.com/trezor/trezor-firmware/pull/4083/commits/6c3e77a795ebca58b8d974d55efbd09e05f121db.